soccer/src/soccer/tutorial/soccer_mom_node_main.cpp

```diff
@@ -0,0 +1,14 @@
+#include <rj_common/network.hpp>
+#include <rj_utils/logging.hpp>
+
+#include "global_params.hpp"
+#include "soccer_mom.hpp"
+
+int main(int argc, char** argv) {
+    rclcpp::init(argc, argv);
+    rj_utils::set_spdlog_default_ros2("processor");
+
+    auto soccer_mom = std::make_shared<tutorial::SoccerMom>();
+    start_global_param_provider(soccer_mom.get(), kGlobalParamServerNode);
```

Comment:
You also probably don't need a param provider for soccer mom, or to do whatever `set_spdlog_default_ros2` does (I'm not even sure), but I don't expect you to know these things yet.